instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -32,19 +32,15 @@
 Limitations
 -----------
 
-Note that calls that do not use the higher-level APIs but use
-:code:`requests.sessions.Session.send` (or an alias thereof) directly, are
-currently not traced. If you find any other way to trigger an untraced HTTP
-request, please report it via a GitHub issue with :code:`[requests: untraced
-API]` in the title.
+If you find any way to trigger an untraced HTTP request, please report it via a
```

Comment:
Wondering if this comment even needs to be in the code? An uninstrumented trace feels like a self-evident issue.